src/main/groovy/com/xebialabs/gradle/xldeploy/DeployTask.groovy

```diff
@@ -39,13 +38,19 @@ class DeployTask extends BaseDeploymentTask {
   @SuppressWarnings("GroovyUnusedDeclaration")
   public void executeDeployment() {
     try {
+      if(xldExtension.xldDeployLogLevel) {
+        logLevel = LogLevel.valueOf(xldExtension.xldDeployLogLevel)
+      }
+      else {
+        logLevel = LogLevel.INFO
+      }
       boot()
       final ConfigurationItem deploymentPackage = uploadPackage();
       environmentId = environmentId ?: xldExtension.xldEnvironmentId
       if (Strings.emptyToNull(environmentId) != null) {
         deployPackage(deploymentPackage)
       } else {
-        logger.info("No environmentId is specified for $DEPLOY_TASK_NAME so skipping deployment")
+        logger.log(logLevel, "No environmentId is specified for $XlDeployPlugin.DEPLOY_TASK_NAME so skipping deployment")
```

Comment:
This will make that all log messages will be send to the log level that has been specified. Making it impossible to distinct between info, warn, debug. The code should decide which level the message goes to. The end user decides which level to see, correct?

Reply:
This change is to be able to make the output to a different level than the default of LogLevel.INFO. Currently if we set it to LogLevel.INFO the amount of information in the log makes it unusable long.  This allows the end user to configure xldDeployLogLevel to a different level.  In our flow when we do a deployment we want the deployment steps to be logged at LogLevel.Lifecycle.  This provides a similar level of logging that the XLDeploy Maven plugin provides.